From aff2636319bd037fbf4a09b7f6a85cf7b863c61b Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
Date: Thu, 28 Jan 2021 13:46:11 +0200
Subject: [PATCH 65/76] lmb: Increase the number of allowed logical memory
 block

Increase MAX_LMB_REGIONS to allow multiple reserved regions in Linux
device tree.

Issue: ALB-6340
Signed-off-by: Ghennadi Procopciuc <Ghennadi.Procopciuc@nxp.com>
---
 include/lmb.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/lmb.h b/include/lmb.h
index 3b338dfee0..a71e3ca812 100644
--- a/include/lmb.h
+++ b/include/lmb.h
@@ -10,9 +10,10 @@
  * Logical memory blocks.
  *
  * Copyright (C) 2001 Peter Bergner, IBM Corp.
+ * Copyright 2021 NXP
  */
 
-#define MAX_LMB_REGIONS 8
+#define MAX_LMB_REGIONS 16
 
 struct lmb_property {
 	phys_addr_t base;
-- 
2.17.1

